tools/test_history.py

```diff
@@ -297,19 +298,19 @@ def parse_args(raw: List[str]) -> argparse.Namespace:
         help='name of the suite containing the test',
     )
     parser.add_argument(
-        'test',
+        '--test',
         help='name of the test',
     )
     parser.add_argument(
-        'job',
-        nargs='*',
+        '--jobs',
         help='names of jobs to display columns for, in order',
-        default=[],
+        default='',
     )
     args = parser.parse_args(raw)
 
-    args.jobs = None if args.all else args.job
-    if args.jobs == []:  # no jobs, and not None (which would mean all jobs)
+    args.jobs = None if args.all else args.jobs.split(',')
+    # We dont allow implicit or empty "--jobs", unless "--all" is specified.
+    if args.jobs and (args.jobs == [] or args.jobs[0] == ''):  
```

Comment:
seems like you have some trailing spaces here my friend :P
```suggestion
    if args.jobs and (args.jobs == [] or args.jobs[0] == ''):
```